{
  "id": "ALSA-2024:4058",
  "summary": "Important: python3.11 security update",
  "affected": [
    {
      "package": {
        "ecosystem": "AlmaLinux:8",
        "name": "python3.11"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "3.11.9-1.el8_10"
            }
          ]
        }
      ]
    },
    {
      "package": {
        "ecosystem": "AlmaLinux:8",
        "name": "python3.11-debug"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "3.11.9-1.el8_10"
            }
          ]
        }
      ]
    },
    {
      "package": {
        "ecosystem": "AlmaLinux:8",
        "name": "python3.11-devel"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "3.11.9-1.el8_10"
            }
          ]
        }
      ]
    },
    {
      "package": {
        "ecosystem": "AlmaLinux:8",
        "name": "python3.11-idle"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "3.11.9-1.el8_10"
            }
          ]
        }
      ]
    },
    {
      "package": {
        "ecosystem": "AlmaLinux:8",
        "name": "python3.11-libs"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "3.11.9-1.el8_10"
            }
          ]
        }
      ]
    },
    {
      "package": {
        "ecosystem": "AlmaLinux:8",
        "name": "python3.11-rpm-macros"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "3.11.9-1.el8_10"
            }
          ]
        }
      ]
    },
    {
      "package": {
        "ecosystem": "AlmaLinux:8",
        "name": "python3.11-test"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "3.11.9-1.el8_10"
            }
          ]
        }
      ]
    },
    {
      "package": {
        "ecosystem": "AlmaLinux:8",
        "name": "python3.11-tkinter"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "3.11.9-1.el8_10"
            }
          ]
        }
      ]
    }
  ],
  "related": [
    "CVE-2023-6597",
    "CVE-2024-0450"
  ],
  "published": "2024-06-24T00:00:00Z",
  "modified": "2024-06-24T12:16:49Z",
  "details": "Python is an interpreted, interactive, object-oriented programming language, which includes modules, classes, exceptions, very high level dynamic data types and dynamic typing. Python supports interfaces to many system calls and libraries, as well as to various windowing systems.\n\nSecurity Fix(es):\n\n* python: Path traversal on tempfile.TemporaryDirectory (CVE-2023-6597)\n* python: The zipfile module is vulnerable to zip-bombs leading to denial of service (CVE-2024-0450)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer the CVE page(s) listed in the References section.",
  "references": [
    {
      "url": "https://access.redhat.com/errata/RHSA-2024:4058",
      "type": "ADVISORY"
    },
    {
      "url": "https://access.redhat.com/security/cve/CVE-2023-6597",
      "type": "REPORT"
    },
    {
      "url": "https://access.redhat.com/security/cve/CVE-2024-0450",
      "type": "REPORT"
    },
    {
      "url": "https://bugzilla.redhat.com/2276518",
      "type": "REPORT"
    },
    {
      "url": "https://bugzilla.redhat.com/2276525",
      "type": "REPORT"
    },
    {
      "url": "https://errata.almalinux.org/8/ALSA-2024-4058.html",
      "type": "ADVISORY"
    }
  ]
}